client/client.go

```diff
@@ -1,92 +0,0 @@
-package client
```

Comment:
i removed it, I am not sure this is useful at all, given that it's not different from the actual client imported.